python/servo/command_base.py

```diff
@@ -130,8 +130,14 @@ def cargo_build_id(self):
             self._cargo_build_id = open(filename).read().strip()
         return self._cargo_build_id
 
+    def get_target_dir(self):
+        if "CARGO_TARGET_DIR" in os.environ:
+            return os.environ["CARGO_TARGET_DIR"]
+        else:
+            return path.join(self.context.topdir, "target")
+
     def get_binary_path(self, release, dev):
-        base_path = path.join("components", "servo", "target")
+        base_path = self.get_target_dir()
```

Comment:
`build_commands.py`, `post_build_commands.py`, and `testing_commands.py` still contain references to the old target directory.